master/templates/conf-template.conf

```diff
@@ -1,10 +1,16 @@
-{% if config.pop('nested', True) %}
-{{ config.name }}:
-  {% for arg, val in config.params.items() %}
-  - {{ arg }}: {{ val }}
-  {% endfor %}
-{% else %}
-{% for arg, val in config.items() %}
-{{ arg }}: {{ val }}
-{% endfor %}
-{% endif %}
\ No newline at end of file
+{%- macro render_yaml(config_dict, offset=2) -%}
```

Comment:
That's the opening for the definition of the macro.

https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#jinja-macros